src/Evolution/DiscontinuousGalerkin/Limiters/MinmodHelpers.cpp

```diff
@@ -54,7 +54,8 @@ BufferWrapper<VolumeDim>::BufferWrapper(const Mesh<VolumeDim>& mesh) noexcept
       });
   contiguous_boundary_buffer_.reset(static_cast<double*>(
       // clang-tidy incorrectly thinks this is a 0-byte malloc
-      // NOLINTNEXTLINE(clang-analyzer-unix.API)
+      // NOLINTNEXTLINE(clang-analyzer-unix.API,
```

Comment:
`NOLINTNEXTLINE` doesn't work if split across multiple lines

Reply:
Sure enough, I agree this doesn't work. Furthermore, I tested a little more and realized that clang-tidy isn't flagging this under the old warning anymore. So the fix is simple: keep the `NOLINTNEXTLINE` but update the warning name. I'll do this here (and in the other instance below) after we settle on the CMakeLists question.